### ISSUE TITLE
Questions

### ISSUE THREAD

**Issue by coke11** [GreenBayRules/sheshank.com#1]
How is this going to be hosted? Have you got the domain yet?

**Comment by Sheshank-s** [comment]
Yeah! I got the domain through GoDaddy. Im working on transfering it!

**Comment by Sheshank-s** [comment]
I really have no clue what to do with this :D @coke11
